edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js

```diff
@@ -5,6 +5,20 @@ var edx = edx || {};
 
     edx.instructor_dashboard = edx.instructor_dashboard || {};
     edx.instructor_dashboard.proctoring = edx.instructor_dashboard.proctoring || {};
+    var examStatusReadableFormat = {
+        eligible: 'Eligible',
+        created: 'Created',
+        ready_to_start: 'Ready to start',
+        started: 'Started',
+        ready_to_submit: 'Ready to submit',
+        declined: 'Declined',
+        timed_out: 'Timed out',
+        submitted: 'Submitted',
+        verified: 'Verified',
+        rejected: 'Rejected',
+        not_reviewed: 'Not reviewed',
+        error: 'Error'
+    };
```

Comment:
I'm thinking we should wrap all of these inside of gettext()'s to make internationalizable.